openmmtools/multistate/multistatesampler.py

```diff
@@ -1551,6 +1551,33 @@ def _offline_analysis(self):
         self._reporter.write_online_data_dynamic_and_static(self._iteration,
                                                             f_k=self._last_mbar_f_k,
                                                             free_energy=(free_energy, self._last_err_free_energy))
+        # Estimate performance
+        # TODO: use units for timing information to easily convert between seconds and days
+        # there are some mcmc_moves that have no timestep attribute, catch exception
+        try:
+            current_simulated_time = self.mcmc_moves[0].timestep*self._iteration*self.mcmc_moves[0].n_steps
+        except AttributeError:
+            current_simulated_time = 0.0 * unit.nanosecond  # Hardcoding to 0 ns
+
+        # There's no timing data dictionary for 1st iteration, catch exception
+        try:
+            partial_total_time = self._iteration*self._timing_data["average_seconds_per_iteration"]
+            performance = current_simulated_time.in_units_of(unit.nanosecond)/(partial_total_time/86400)
+            performance = performance.value_in_unit(unit.nanosecond)
+        except KeyError:
+            performance = None
+        # Write real time offline analysis YAML file
+        # TODO: Specify units
+        data_dict = {"iteration": self._iteration,
+                     "percent_complete": self._iteration*100/self.number_of_iterations,
+                     "mbar_analysis": {"free_energy_in_kT": float(free_energy),
+                                       "standard_error_in_kT": float(self._last_err_free_energy),
+                                       "uncorrelated_samples": float(analysis._equilibration_data[-1])
```

Comment:
I'd use `number_of_uncorrelated_samples`.

Is there anything else in `_equilibration_data` we could report too, like the initial production iteration or statistical inefficiency?